Только в /root/dk_scripts/tmp/1: 9-Oct-2013022817LINK-I-Upfa48KTCS4-SW1-48_192.168.22.71.conf
diff -r /root/dk_scripts/tmp/2/TCS0-SW1-24_192.168.22.62.conf /root/dk_scripts/tmp/1/TCS0-SW1-24_192.168.22.62.conf
133a134,136
> interface fastethernet12
>  switchport trunk native vlan 66 
> !
Только в /root/dk_scripts/tmp/2: TCS4-SW1-48_192.168.22.71.conf
